doc/usage.rst

```diff
@@ -458,10 +459,15 @@ use :meth:`pooch.Pooch.load_registry` to read them:
         # We'll load it from a file later
         registry=None,
     )
-    GOODBOY.load_registry(os.path.join(os.path.dirname(__file__), "registry.txt"))
-
-The ``registry.txt`` file in this case is in the same directory as the ``datasets.py``
-module and should be shipped with the package. It's contents are:
+    # Get registry file from package_data
+    registry_file = pkg_resources.resource_stream("plumbus", "registry.txt")
+    # Load this registry file
+    GOODBOY.load_registry(registry_file)
+
+The ``registry.txt`` file in this case is in the ``plumbus/`` directory module and should be
```

Comment:
```suggestion
The ``registry.txt`` file is in the ``plumbus/`` directory (the package) and should be
```

Reply:
this repeats "the package"

Reply:
Right, didn't see that one in the next line. Just feel like "directory module" sounds strange.